### DISCUSSION TITLE
fails to load VST fx on Ableton 11

### DISCUSSION THREAD

**Discussion by squalle10** [apohl79/audiogridder#621] [Help]
Hello, I tried the plugin, I can load the Vst on Fl studio but on Ableton 11 it's not working every plugin failed to load.
I don't understand why ? @
![Capture d’écran 2021-06-21 à 18 23 29](https://user-images.githubusercontent.com/86251796/122795653-d3437f00-d2bd-11eb-8ab3-0dd05d294d71.jpg)

**Comment by CaBuffalo** [comment]
Hi, there,

This has been reported. We will work on it.

Thanks.

On Mon, Jun 21, 2021 at 9:24 AM squalle10 ***@***.***> wrote:

> Hello, I tried the plugin, I can load the Vst on Fl studio but on Ableton
> 11 it's not working every plugin failed to load.
> I don't understand why ? @
> [image: Capture d’écran 2021-06-21 à 18 23 29]
> <https://user-images.githubusercontent.com/86251796/122795653-d3437f00-d2bd-11eb-8ab3-0dd05d294d71.jpg>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/621>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CLWZWCZZHOGBV6FCTTT5RTRANCNFSM47B5E7GA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by squalle10** [reply]
oh thanks
